wsi_deid/import_export.py

```diff
@@ -110,9 +110,32 @@ def getSchema():
     # that contain a single file that parses as valid json, then we need to
     # load those files and combine them into a single schema in lieu of using
     # this default SCHEMA_FILE_PATH
-    logger.info('Using %s (length %d) for schema' % (
-        SCHEMA_FILE_PATH, os.path.getsize(SCHEMA_FILE_PATH)))
-    return json.load(open(SCHEMA_FILE_PATH))
+    schemaFolderId = Setting().get(PluginSettings.WSI_DEID_SCHEMA_FOLDER)
+    mergedSchema = {}
+
+    if schemaFolderId:
+        schemaFolder = Folder().load(schemaFolderId, force=True)
+
+        for filepath, file in Folder().fileList(schemaFolder, data=False):
+            currentObject=json.load(File().open(file))
```

Comment:
This json load source be guarded in a try / except in case the file ISN'T json.  (and if it fails, we just ignore that file).  We probably want to check to make sure the file is less than some maximum size (a few megabytes), too, as otherwise it can exhaust memory.

Lastly, we might want to use the yaml loader (but this could be done in some future PR), as yaml allows json as a subset and this would give greater flexibility to the process.

Reply:
addressed in 2a93feaf7fb521edcaeb54d9812d4320d10df69e